main.tf

```diff
@@ -38,7 +38,8 @@ module load_balancer {
 ############
 
 module container_definition {
-  source = "./modules/fargate_container_definition"
+  source  = "transcend-io/fargate-container/aws"
```

Comment:
can u integration this new repo -- https://www.notion.so/transcend/Creating-a-new-github-repo-81c79c1723414d4b96f89f3eb0324695